src/main/java/study/step_1/domain/Calculator.java

```diff
@@ -0,0 +1,154 @@
+package study.step_1.domain;
+
+import static study.step_1.constant.error.ErrorMessage.*;
+import static study.step_1.domain.DelimConstant.*;
+
+public class Calculator implements Calculate {
+
+    public static final int INVALID_INPUT_VALUE = 0;
+    public static final String USING_PREFIX = "//";
+    public Character customDelimiter;
+
+    public String expression;
+
+    public Calculator(String expression) {
+        if (BlankAndNull(expression)) return;
```

Comment:
```suggestion
        if (BlankAndNull(expression)) {
            return;
        }
```

자바에서는 단 한줄이라도 꼭 중괄호를 넣어주세요. (자바 컨벤션)